Cargo.toml

```diff
@@ -25,33 +25,41 @@ log = "0.4.14"
 nix = { version = "0.26", optional = true }
 oci-distribution = "0.9.4"
 oci-spec = "0.5.8"
-ocicrypt-rs = { git = "https://github.com/confidential-containers/ocicrypt-rs", rev = "52c9a3c", default-features = false, features = ["keywrap-jwe", "async-io"], optional = true }
+ocicrypt-rs = { git = "https://github.com/confidential-containers/ocicrypt-rs", rev = "453c699", default-features = false, features = ["keywrap-jwe", "keywrap-keyprovider-cmd", "keywrap-keyprovider-grpc", "keywrap-keyprovider-native", "keywrap-keyprovider-ttrpc", "async-io"], optional = true }
 prost = { version = "0.11", optional = true }
 sequoia-openpgp = { version = "1.7.0", default-features = false, features = ["compression", "crypto-rust", "allow-experimental-crypto", "allow-variable-time-crypto"], optional = true }
+protobuf = { version = "3.2.0", optional = true }
 serde = { version = ">=1.0.27", features = ["serde_derive", "rc"] }
 serde_json = ">=1.0.9"
 serde_yaml = { version = "0.8", optional = true }
 sha2 = ">=0.10"
+
 # Latest version of sigstore conflicts with sequoia-openpgp, so stick to v0.3
 sigstore = { version = "0.3.0",  optional = true}
 strum = { version = "0.24", features = ["derive"] }
 strum_macros = "0.24"
 tar = "0.4.37"
-tokio = "1"
+tokio = {version = "1.0", features = ["full"]}
```

Comment:
Please help to explicitly specify features needed:)

Reply:
Nice suggestion, I'm going to